pypeit/tests/old_test_objfind.py

```diff
@@ -1,9 +1,4 @@
 # Module to run tests on ararclines
```

Comment:
Should we get rid of this file or adapt it to test the existing object finding algorithms...